README.md

```diff
@@ -1 +1,40 @@
-# Hello-Microverse-Reviewable
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> A basic Hello Microverse Project with HTML and CSS. It has Linters Implemented and ignored redundant files using .gitignore
+
+![screenshot]()
+
+
```

Comment:
## Changes required:

⚠️ The screenshot for your project is not showing. Kindly add it to your files. It will make your README look much more professional!